### PR TITLE
pages.pt_BR/linux/*: add Portuguese Brazilian translations

### PR DESCRIPTION
**Pull request by rffontenelle**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by rffontenelle** [comment]
@sebastiaanspeck changed the PR name. Thanks.

**Comment by rffontenelle** [comment]
Oh ok. I didn't realized you had it renamed so I was puzzling around look for what was wrong. I'll be more careful with the style next PRs

File: pages.pt_BR/linux/apport-bug.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# apport-bug
+
+> Registrar um relatório de bug no Ubuntu.
```
Comment (renie) [inline pages.pt_BR/linux/apport-bug.md:3]:
```suggestion
> Registra um relatório de bug no Ubuntu.
```

File: pages.pt_BR/linux/cgclassify.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# cgclassify
+
+> Mover tarefas em execução para `cgroups`.
```
Comment (renie) [inline pages.pt_BR/linux/cgclassify.md:3]:
```suggestion
> Move tarefas em execução para `cgroups`.
```

File: pages.pt_BR/linux/cgcreate.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# cgcreate
+
+> Criar cgroups, usados para limitar, medir e controlar recursos usados pelos processos.
```
Comment (renie) [inline pages.pt_BR/linux/cgcreate.md:3]:
```suggestion
> Cria cgroups, usados para limitar, medir e controlar recursos usados pelos processos.
```

File: pages.pt_BR/linux/cgexec.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# cgexec
+
+> Limitar, medir e controlar recursos usados pelos processos.
```
Comment (renie) [inline pages.pt_BR/linux/cgexec.md:3]:
```suggestion
> Limita, mede e controla recursos usados pelos processos.
```

File: pages.pt_BR/linux/ip-route-show.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
+# ip route show
+
+> Exibir subcomando para o gerenciamento de tabelas de roteamento de IP.
```
Comment (renie) [inline pages.pt_BR/linux/ip-route-show.md:3]:
```suggestion
> Exibe subcomando para o gerenciamento de tabelas de roteamento de IP.
```

File: pages.pt_BR/linux/megatools-dl.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+# megatools-dl
+
+> Baixar arquivos de `mega.nz`.
```
Comment (renie) [inline pages.pt_BR/linux/megatools-dl.md:3]:
```suggestion
> Baixa arquivos de `mega.nz`.
```

**Review by renie — CHANGES_REQUESTED** [review]
Just small fixes on some verbs to not use infinitive on the description of commands.

**Review by renie — APPROVED** [review]
Now all seems good! Thanks for one more contribution! :sun_with_face:

**Pull request merged**